ui/README.md

```diff
@@ -121,20 +128,33 @@ If successful, you will see bridge processes run when you issue a command. For e
 
 6. Keep the bridge processes running. Open a separate terminal window and go to the `sokol-kovan-bridge` folder to install and unpack this repository.
 
-    * `git clone https://github.com/poanetwork/tokenbridge`  
-    * `cd tokenbridge`
-    * Update submodules  
-`git submodule update --init --recursive` 
-    * Install dependencies  
-`yarn install`
-    * Go to UI sub-repository  
-`cd ui`
+  * Clone the repository
+
+```
+git clone --recursive https://github.com/poanetwork/tokenbridge && cd tokenbridge
+```
+
+  * Install dependencies and compile Smart Contracts
+
+```
+yarn install && yarn install:deploy && yarn compile:contracts
+```
+  * Go to UI sub-repository  
+```
+cd ui
+```
 
 _**Note**: The bridge UI configuration should be performed with an unprivileged Linux account or with the following flag `npm install --unsafe-perm`. [More information](https://docs.npmjs.com/misc/scripts#user)_
```

Comment:
This is not needed in this section. Similar note should be provided in the step 5 if it make sense for `yarn`.

Reply:
Fixed, please see [here](https://github.com/poanetwork/tokenbridge/pull/75#issuecomment-496453969)